ffi/CMakeLists.txt

```diff
@@ -37,9 +37,11 @@ add_library(llvmlite SHARED assembly.cpp bitcode.cpp core.cpp initfini.cpp
             passmanagers.cpp targets.cpp dylib.cpp linker.cpp object_file.cpp)
 
 # Find the libraries that correspond to the LLVM components
-# that we wish to use
-# llvm_map_components_to_libnames(llvm_libs support core irreader)
-llvm_map_components_to_libnames(llvm_libs all)
+# that we wish to use.
+# The following line is broken with LLVM 10.0.0 due to a potential bug in
+# the LLVM cmake setup, so we use the workaround instead.
+# BROKEN: llvm_map_components_to_libnames(llvm_libs all)
```

Comment:
Let's be specific, what precisely was the problem, how did it manifest, which platforms was it on etc. Easier to know if it's fixed if we know how to reproduce/what to look for.

Reply:
probably should reference a bug-report here.

Reply:
https://bugs.llvm.org/show_bug.cgi?id=47003 <--- added link to source